recipe/meta.yaml

```diff
@@ -1,58 +1,79 @@
 {% set name = "anaconda-cloud-auth" %}
-{% set version = "0.5.1" %}
+{% set version = "0.7.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anaconda_cloud_auth-{{ version }}.tar.gz
-  sha256: 23b428b13e88c036905dc266a4b3c1dc35565e706341478659f316e45cd6c4dd
+  sha256: bdb49cdb5a22b5d0c1d825cf2b0529005628fa987b5a1466283d80722590044a
 
 build:
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
     - hatchling
     - hatch-vcs >=0.3
     - pip
+    - setuptools-scm >=7.1
   run:
     - python
+    - anaconda-cli-base >=0.4.0
+    - cryptography >=3.4.0
     - keyring
     - pkce
-    - python-dotenv
     - pydantic
     - pyjwt
+    - python-dotenv
     - requests
-    - cryptography >=3.4.0
     - semver <4
   run_constrained:
     - conda >=23.9.0
 
 test:
   imports:
     - anaconda_cloud_auth
+  source_files:
+    - tests
+  requires:
+    - pip
+    - pytest
+    - pytest-mock
+    - responses
+    - types-requests
   commands:
     - pip check
+    # Check if the ANACONDA_CLI_FORCE_NEW environment variable is required with a new release version.
+    - export ANACONDA_CLI_FORCE_NEW=1  # [unix]
+    - set ANACONDA_CLI_FORCE_NEW=1     # [win]
+    - anaconda -h
+    - anaconda -V
+    - anaconda cloud --help
+    - anaconda cloud -V
     - python -c "from anaconda_cloud_auth import __version__; assert __version__ == \"{{ version }}\""
-  requires:
-    - pip
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - ls -ld /root  # [linux]
```

Comment:
what does this test? a comment would be useful. (should it do `| grep` something?)

Reply:
It was debugging code. I removed it. Thanks for catching it